src/OpenSage.Game/Logic/Player.cs

```diff
@@ -83,7 +83,7 @@ public class Player : IPersistableObject
 
     public AIPlayer? AIPlayer { get; private set; }
 
-    public AcademyStats AcademyStats { get; } = new AcademyStats();
+    public readonly AcademyStats AcademyStats;
```

Comment:
Was there a reason this had to be changed from a property to a field? Is that just in case we choose to persist it in the future?

Reply:
I used a couple of days old `master` as the base, and in that version there was no `AcademyStats` field / class yet, so I added it as a field. Tim had added a stub implementation in the mean time, so there was a conflict when rebasing. I solved it by just taking my changes (since the initialisation had to be moved to the constructor anyway because of `IGame` parameter). I can change it back to property; no strong feelings either way 😄